047-TrafficControlWithDapr/Student/Challenge-07.md

```diff
@@ -0,0 +1,61 @@
+# Challenge 7 - Add secrets management
+
+[< Previous Challenge](./Challenge-06.md) - **[Home](../README.md)** - [Next Challenge >](./Challenge-08.md)
+
+## Introduction
+
+In this challenge, you're going to add the Dapr **secrets management** building block.
+
+## Description
+
+Almost all non-trivial applications need to _securely_ store secret data like API keys, database passwords, and more. As a best practice, secrets should never be checked into the version control system. But, as the same time,they need to be accessible to code running in production. This is generally a challenging requirement, but critical to get right.
```

Comment:
Should say "But at the same time, they need to be accessible to code running in production" (replaced "as" with "at" and removed extra comma)